### DISCUSSION TITLE
ByteOrder Crate, Reading Endeaness from Buffer

### DISCUSSION THREAD

**Discussion by sajvn0x** [obhq/exfat#8] [General]
![big_endian](https://user-images.githubusercontent.com/108508613/223156492-59c61fef-5d3e-4f47-8608-84aa717b4019.png)

I tried for a long time to decode how byteorder crate read from buffer to Endeaness number
Unfortunately, I failed to decode it. Can you give me a brief explanation about it?

**Comment by ultimaweapon** [comment]
517 is the correct output for bytes `0x0205` in big-endian. What is your expected output?

**Reply by sajvn0x** [reply]
Ok, Thanks I Understood now, We need to consider the first 2 elements as hex values right, I actually considered them as decimals. now ok
